alembic/versions/8d80e867d3dd_reseed_db.py

```diff
@@ -0,0 +1,66 @@
+"""Reseed DB
+
+Revision ID: 8d80e867d3dd
+Revises: b557abac858b
+Create Date: 2020-07-24 18:32:20.914310
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table, column
+from sqlalchemy import String, Integer, Date, Unicode, Boolean
+
+# revision identifiers, used by Alembic.
+revision = '8d80e867d3dd'
+down_revision = 'b557abac858b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute('''TRUNCATE TABLE "gardens"; ''')
+    op.execute('''TRUNCATE TABLE "plants" CASCADE; ''')
```

Comment:
CASCADE was cool, and its nice to know we had to do this since we CREATED A RELATIONSHIP! between our tables